webapp/src/language/es.json

```diff
@@ -180,6 +180,9 @@
 
   "profile": {
     "generalInformation": "Información General",
+    "eosRates": "Clasificación EOS",
```

Comment:
Clasificación o Calificación? Is that right?

Reply:
Calificación